### ISSUE TITLE
Piratebay search not working

### ISSUE THREAD

**Issue by eSoares** [Flexget/Flexget#2702]
### Expected behaviour:
Search on piratebay to return results.

### Actual behaviour:
No results found.

### Problem details
This is happening due to recent changes on piratebay where they force the browser to execute javascript before displaying the results. (This is verifiebly by running a `wget -O - https://pirateproxy.surf/search.php\?q\=linux\&cat\=0`, output bellow)

There are a couple of possible solutions:
- Using Selenium #2372
- Using an external lib to handle the kirks of pirate bay, like https://github.com/vikstrous/pirate-get
  
#### Log:

```
2020-06-18 15:47:18 VERBOSE  discover      tv-discover     Searching for `<>` with plugin `piratebay` (1 of 1)
2020-06-18 15:47:18 DEBUG    piratebay     tv-discover     Using https://thepiratebay.org/search/<>/0/7/0 as piratebay search url
2020-06-18 15:47:18 DEBUG    utils.requests tv-discover     GETing URL https://thepiratebay.org/search/<>/0/7/0 with args () and kwargs {'allow_redirects': True, 'timeout': 30}
2020-06-18 15:47:20 DEBUG    piratebay     tv-discover     Using https://thepiratebay.org/search/<>/0/7/0 as piratebay search url
2020-06-18 15:47:20 DEBUG    utils.requests tv-discover     GETing URL https://thepiratebay.org/search/<>/0/7/0 with args () and kwargs {'allow_redirects': True, 'timeout': 30}
2020-06-18 15:47:21 DEBUG    discover      tv-discover     No results from piratebay
```

wget result to pirate bay:
```html
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>The Pirate Bay - The galaxy's most resilient bittorrent site</title>
<link href="/static/normalize.css" rel="stylesheet" type="text/css">
<script type="text/javascript" src="https://www.powr.io/powr.js?external-type=html"></script>

<link href="/static/tpb.css" rel="stylesheet" type="text/css">
<script src="/static/main.js"></script>
<script src="/static/tinysort.min.js"></script>
</head>
<body id="browse" onload="jswarnclear()">
<main>
<b><font color="RED"><label id="jscrwarn">Enable JS in your browser!</label></font></b>
<script>document.getElementById("jscrwarn").innerHTML='';</script>
<b><font color="RED"><label id="jscrwarn2">Your adblock may block important javascript components, check that main.js is loaded or the webpage won't work.</label></font></b>
...
</body>
</html>
```

**Comment by chestm007** [comment]
I had this problem, switched over to using 1337x

**Comment by ksurl** [comment]
There's a pr 2713 that is supposed to fix it

**Comment by kristenmills** [comment]
This was fixed in #2713 and #2745. 

Will be in the next flexget release.
